Cogs/ServerManagement.py

```diff
@@ -339,3 +341,48 @@ async def on_button_click(self, res):
                 else:
                     await member.add_roles(role)
                     await res.respond(type=InteractionType.ChannelMessageWithSource, content=f'Gave you the {role.name} role!')
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        """
+        Checks if the author of the message was a bot and if it was sent in a correct channel
+        Looks through a message that has been sent to see if it was a question
+        If the message had a ? in it it replies to it with "That looks interesting"
+        """
+
+        # If the author is a bot return
+        if message.author.bot:
+            return
+
+        # Checks if the channel is a requested faq channel
+        if message.channel in self.faq_channels:
+
+            # If there is a ? in the content reply "That looks interesting" and break out of the loop
+            if '?' in message.content:
+                await message.reply("That looks interesting")
+    
+    @commands.command()
+    @commands.has_permissions(administrator=True)
+    async def faq(self, ctx):
+        """
+        Allows/disallows the faq machine learning feature for a specific channel
+        """
+
+        
+        
+
+        # If the channel is in the list remove it and return
+        if ctx.channel in self.faq_channels:
+            self.faq_channels.remove(ctx.channel)
+            await ctx.reply("FAQ has been disabled for this channel!")
+
+            # Logging
+            await log(self.bot, f'{ctx.author} has disabled FAQ for the {ctx.channel} channel')
+            return
+        
+        # If the channel is not in the list add it to the end
+        await ctx.reply("FAQ has been enabled for this channel!")
+        self.faq_channels.append(ctx.channel)
+
+        # Logging
+        await log(self.bot, f'{ctx.author} has enabled FAQ for the {ctx.channel} channel')
```

Comment:
Remember to have an empty space at the end of your files.

Reply:
Empty space has been added

Reply:
Still looks weird :man_shrugging: